provisioning_templates/provision/kickstart_default.erb

```diff
@@ -39,7 +39,11 @@ This template accepts the following parameters:
   section_end = (rhel_compatible && os_major <= 5) ? '' : '%end'
 %>
 install
+<% if host_param('live_img') %>
+liveimg --url=<%= host_param('live_img') %> <% unless proxy_string.empty? %> --proxy=<%= proxy_string %><% end %>
```

Comment:
I'd rather see the same param name here not to confuse users: `host_param('live_img')`

Reply:
Yeah I actually edited my comment, I mispasted it. Agreed.

Reply:
done